subprojects/gradle/qapps/src/main/kotlin/com/avito/plugin/QAppsUploadAction.kt

```diff
@@ -34,10 +34,16 @@ internal class QAppsUploadAction(
         Retrofit.Builder()
             .baseUrl(host)
             .addConverterFactory(GsonConverterFactory.create(GsonBuilder().create()))
-            .client(httpClient())
+            .client(
+                httpClientProvider.provide(
+                    "qapps",
+                    TimeUnit.SECONDS.toMillis(60),
```

Comment:
We can use java.time.Duration. As for me, it looks better, not inversed:

```kotlin
TimeUnit.SECONDS.toMillis(60)

Duration.ofSeconds(60)
```

Reply:
it is , but we still don't support java8 api's

Reply:
Can't we use it inside Gradle plugins?

Reply:
HttpClientProvider also provide for android runtime